articles/vercel-preview/index.mdx

```diff
@@ -41,33 +118,44 @@ jobs:
         run: npm ci && npx playwright install --with-deps
       - name: Run Playwright tests
         run: npx playwright test
+        # 👉 Add this variables
         env:
           BASE_URL: ${{ github.event.deployment_status.environment_url }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           ARGOS_BRANCH: ${{ github.event.deployment_status.environment == 'Production' && 'main' || '' }}
 ```
 
-For a comprehensive explanation of each step, consult the [Argos documentation](https://argos-ci.com/docs/run-on-preview-deployment).
+For a comprehensive explanation of each step, read [Argos documentation](https://argos-ci.com/docs/run-on-preview-deployment).
```

Comment:
I don't want to have to read another article, just explain me what you are doing here.